javacord-api/src/main/java/org/javacord/api/interaction/SlashCommandInteractionOptionsProvider.java

```diff
@@ -168,11 +168,11 @@
      *
      * @return An Optional with the double value of such an option if it exists; an empty Optional otherwise
      * @deprecated Use {@link #getOptions() getOptions().get(0).getNumberValue()}
```

Comment:
Won't mark every place but the same should be done for `Second` and `Third` NumberValue
```suggestion
     * @deprecated Use {@link #getOptions() getOptions().get(0).getDecimalValue()}
```